Jessbumblebee/firstissue

jessbumblebee opened the pull request:
### Description

Add a description field to the CiviForm all programs page and adapt change "All Programs" to "Program Dashboard" as per the mocks. The intention is to make it easier for civi form admins to understand what this page is for. 

Additional notes I skipped internationalization based on our conversation about this not being a priority for admins
I also did not add tests based on the instructions in the testing documentation which mentions that it is up to our discretion if smaller changes to views should have unit tests.

## Release notes:

Rename the "All programs" page to "Program dashboard" and add a description to make it easier for Civi Form admins to understand the purpose of the page.

### Checklist

- [ x ] Added the correct label: < feature | bug | dependencies | infrastructure | ignore-for-release >
- [ x ] Created tests which fail without the change (if possible)
- [ x ] Extended the README / documentation, if necessary

### Issue(s)

Fixes #3129

nbeloglazov commented:
Few comments on PR
1. Change the name of PR from `Jessbumblebee/firstissue` to more meaningful message. You can do it from github UI.
2. For checklist don't add spaces between [ and x. It should be [x] to render as checkbox.
3. I think you change affects more screenshots. Basically all screenshots with program index should be updated. Also looks like there is some conflict with publish-modal. So you should merge `main` branch to your branch and rerun tests.

nbeloglazov reviewed (changes requested):
Overall looks good! Expecting more screenshots to be added. Once you are ready - request review again in github UI.

jessbumblebee commented:
Hi Mikita

Thanks for the comments. I fixed some formatting issues and cleaned  up my git branch which I had messed up a little.
I can't find any additional tests that fail. Let me know if there is anything additional that I should other than the standard browser tests to find some.

Quick question: is there a naming convention for PRs we use such as naming issues by the branch name? I initially went for a descriptive name in the sense of "it describes what the change does" and would like to understand your comment around better naming so that I can apply it next time.

Thanks 

Jessica
